### PR TITLE
Update Email Provider Database with the "163.com" entry

### PR DESCRIPTION
**Pull request by iscas-zac**
Hello, I'm very interested in this app and tried it with a newly created 163 account (while my other 126 accounts has no spare auth codes). It seems that your descriptions are out of date for the provider, so I updated it a little. I'm not very familiar with your guideline here, so adjust the wording as you like.

By the way, while the e-mail system is the only mature decentralized system and apt for the chat data communication, I'm really curious whether is there any way to directly chat between two clients without the server (which is really what p2p means), considering the complicated network factors like IPv4/NAT, proxy, firewall, ad-hoc wireless network connection and so on. Are there any progress for the mission impossible? Thanks.

### CODE REVIEW

File: _providers/163.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ server:
   socket: SSL
   hostname: smtp.163.com
   port: 465
-last_checked: 2022-06
+last_checked: 2024-11
```
Comment (r10s) [inline _providers/163.md:15]: `PREPARATION` always requires a `before_login_hint`, sth. as:

```suggestion
last_checked: 2024-11
before_login_hint: Enable "POP3/SMTP/IMAP" on the website, add a third-party auth code and use that as the login password
```

below the text a "Read More" link will be shown automatically. that link then opens https://providers.delta.chat/163

**Comment by r10s** [comment]
> By the way, while the e-mail system is the only mature decentralized system and apt for the chat data communication, I'm really curious whether is there any way to directly chat between two clients without the server (which is really what p2p means)

you have luck 🍀

yesterday, we introduced ["realtime Peer-to-Peer networking"](https://delta.chat/en/2024-11-20-webxdc-realtime) for webxdc apps - and one of the demo apps is a chat app :)

https://delta.chat/en/2024-11-20-webxdc-realtime#live-chat-realtime-chat-in-a-chat-

**Comment by iscas-zac** [comment]
> yesterday, we introduced ["realtime Peer-to-Peer networking"](https://delta.chat/en/2024-11-20-webxdc-realtime) for webxdc apps

Good, I'll read it more carefully and try out. That page seems to focus on the ephemeral side of this app, and I am not aware whether it still requires the mail server or not.

> PREPARATION always requires a before_login_hint, sth. as:

I have not reviewed change before. Does this require my commits, or you will handle this alone?

**Comment by r10s** [comment]
if you think, my suggestion makes sense, i will merge that and then this PR

**Comment by iscas-zac** [comment]
Of course it makes more sense than my pure intuition, as I have no idea about the database formats. Thanks for reviewing.

**Review by missytake — APPROVED** [review]

**Comment by missytake** [comment]
Thanks!

**Pull request merged**